### PR TITLE
Feat/verifikasi request

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

1. add upload integration API notes
2. add upload file API
3. change format upload file, notes and update notes

title: integration API from add file, upload and make dynamic form upload
project: Digiteam Inventaris Platform
participants: @rayfajars @yoslie @doohanas @Ibwedagama

### CODE REVIEW

File: src/components/RequestComponents/DetailRequest.vue [hunk 1]
```diff
@@ -93,6 +108,13 @@ export default {
         this.detailRequest.username === this.$store.state.user.profile.name
       );
     },
+    btnPengecekanKelayakan() {
```
Comment (Ibwedagama) [inline src/components/RequestComponents/DetailRequest.vue:111]: minor:

hello @rayfajars maybe we can rename the function to `btnFeasibleCheck` or `btnQualityCheck` just to make the function naming consistent? 🤔

**Review by Ibwedagama — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: src/components/RequestComponents/DetailRequest.vue (hunk 1 above)
Comment (rayfajars) [reply src/components/RequestComponents/DetailRequest.vue]: sorry bli,

for my unconsistent,

i am change  [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/22/commits/36b52e650de6f3f2f092ec512f92a37eb6c10169) bli

**Review by Ibwedagama — APPROVED** [review]

File: src/components/RequestComponents/FormVerifikasiRequest.vue [hunk 2]
```diff
@@ -70,29 +96,61 @@ export default {
...
+        const isValidFormat = [
+          "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+          "application/vnd.ms-excel",
+        ].includes(this.$refs.file.files[0].type);
+        if (isValidFormat) {
+          this.setFile(this.$refs.file.files[0]);
+        } else {
+          this.$store.dispatch("sweetalert/errorAlert", {
+            title: "File ini tidak didukung!",
+            text: "Silahkan ganti file dengan format yg sesuai!",
```
Comment (yoslie) [inline src/components/RequestComponents/FormVerifikasiRequest.vue:114]: **minor**: wording `silakan` not `silahkan`
**discuss**: maybe we can add more information such as typeof file that support this feature?

**Review by yoslie — APPROVED** [review]

File: src/components/RequestComponents/FormVerifikasiRequest.vue (hunk 2 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormVerifikasiRequest.vue]: i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/22/commits/32a9520f6d7684560d3168b3a6eacbabc87549a8)

**Pull request merged**